src/Components/Search/NewSearch/Mobile/SearchResultsList.tsx

```diff
@@ -0,0 +1,247 @@
+import { SearchResultsList_viewer$data } from "__generated__/SearchResultsList_viewer.graphql"
+import { extractNodes } from "Utils/extractNodes"
+import {
+  NewSuggestionItem,
+  SuggionItemOptionProps,
+} from "Components/Search/NewSearch/SuggestionItem/NewSuggestionItem"
+import { Flex, Spinner } from "@artsy/palette"
+import { SystemQueryRenderer } from "System/Relay/SystemQueryRenderer"
+import { SearchResultsListQuery } from "__generated__/SearchResultsListQuery.graphql"
+import { useTracking } from "react-tracking"
+import * as DeprecatedSchema from "@artsy/cohesion/dist/DeprecatedSchema"
+import {
+  SearchNodeOption,
+  formatOptions,
+} from "Components/Search/NewSearch/utils/formatOptions"
+import { SeeFullResults } from "Components/Search/NewSearch/Mobile/SearchResultsList/SeeFullResults"
+import { InfiniteScrollSentinel } from "Components/Search/NewSearch/Mobile/SearchResultsList/InfiniteScrollSentinel"
+import { ContentPlaceholder } from "Components/Search/NewSearch/Mobile/SearchResultsList/ContentPlaceholder"
+import { NoResults } from "Components/Search/NewSearch/Mobile/SearchResultsList/NoResults"
+
+interface SearchResultsListProps {
+  relay: RelayPaginationProp
+  viewer: SearchResultsList_viewer$data
+  query: string
+  onClose: () => void
+}
+
+const ENTITIES_PER_SCROLL = 10
+
+const SearchResultsList: FC<SearchResultsListProps> = ({
+  relay,
+  viewer,
+  query,
+  onClose,
+}) => {
+  const tracking = useTracking()
+  const [isLoading, setIsLoading] = useState(false)
+
+  const options = extractNodes(viewer.searchConnection)
+
+  if (!relay.isLoading() && options.length === 0) {
+    return <NoResults query={query} mt={50} mx={20} />
+  }
+
+  tracking.trackEvent({
+    action_type:
+      options.length > 0
+        ? DeprecatedSchema.ActionType.SearchedAutosuggestWithResults
+        : DeprecatedSchema.ActionType.SearchedAutosuggestWithoutResults,
+    query: query,
+  })
+
+  const formattedOptions: SuggionItemOptionProps[] = formatOptions(
+    options as SearchNodeOption[]
+  )
+
+  const handleLoadMore = () => {
+    if (!relay.hasMore() || relay.isLoading()) {
+      return
+    }
+
+    setIsLoading(true)
+
+    relay.loadMore(ENTITIES_PER_SCROLL, err => {
+      if (err) {
+        console.error(err)
+      }
+
+      setIsLoading(false)
+    })
+  }
+
+  const handleOnRedirect = () => {
+    onClose()
+  }
+
+  return (
+    <>
+      {formattedOptions.map((option, index) => {
+        return (
+          <NewSuggestionItem
+            query={query}
+            option={option}
+            onRedirect={handleOnRedirect}
+            key={index}
+          />
+        )
+      })}
+
+      {relay.hasMore() && <InfiniteScrollSentinel onNext={handleLoadMore} />}
+
+      {!relay.hasMore() && (
+        <SeeFullResults query={query} onClick={handleOnRedirect} />
+      )}
+
+      {isLoading && (
+        <Flex width="100%" minHeight="60px" alignItems="center">
+          <Spinner position="relative" />
```

Comment:
By default Spinner's position is `absolute`. It will look like this:

![Screenshot 2023-05-17 at 10 49 34](https://github.com/artsy/force/assets/3934579/6e238c50-b0b0-4dc5-8ddf-43f46cfc364f)